### ISSUE TITLE
Backticks inserted into render if using $

### ISSUE THREAD

**Issue by 0atman** [MSzturc/obsidian-advanced-slides#193]
**Describe the bug**
There's some escaping bug happening when using dollar `$` signs in a code block.
There are backticks inserted into the rendered presentation, both in the in-obsidian view and the browser view.

**Samples to Reproduce**
![Screenshot_20230126_160126](https://user-images.githubusercontent.com/114097/214895865-393fc3d3-c458-4f81-a91b-546f12d7dc01.png)

**Expected behavior**
I expect the code block to look exactly as it does in Obsidian

Isn't it weird? Thank you!

**Comment by MSzturc** [comment]
Yup... this is a perfect example of not going the short route in software development... the whole escaping code has to be reviewed and has to be simplified... I will do this after 1.20 release.. no more quick fixes here ;-)
